cmake/ProjectsTagsDevel.cmake

```diff
@@ -0,0 +1,12 @@
+set(ICUB_TAG devel)
+set(RTF_TAG devel)
+set(WBToolbox_TAG feature/newFactoryAndNewCMakeStructure)
+set(YARP_TAG devel)
+set(codyco-modules_TAG devel)
+set(icub-tests_TAG devel)
+set(iDynTree_TAG devel)
+set(firmware-shared_TAG devel)
```

Comment:
`icub-` missing ?

Reply:
> Any reason not to keep WB-Toolbox in devel ?

yes, the selected branch `feature/newFactoryAndNewCMakeStructure` is on top and ahead of the `devel` branch. @diegoferigo confirmed that it has the latest fixes working with the latest version of YARP.
@diegoferigo is something preventing you from merging the branch to `devel`?

Reply:
> icub- missing ?

sorry I don't understand. The repo name you mention is incomplete..

Reply:
Sorry, I meant: I guess this line should be `set(icub-firmware-shared_TAG devel)`

Reply:
ah yes absolutely.

Reply:
@nunoguedelha Not really. Considering the change I applied I was waiting some feedback from the iCub setup but no one complained so far. I will merge it soon.